datahub-web-react/src/app/entity/domain/DataProductsTab/DataProductBuilderForm.tsx

```diff
@@ -47,6 +73,37 @@ export default function DataProductBuilderForm({ builderState, updateBuilderStat
             <Form.Item label={<Typography.Text strong>Description</Typography.Text>}>
                 <StyledEditor doNotFocus content={builderState.description} onChange={updateDescription} />
             </Form.Item>
+            <Collapse ghost>
```

Comment:
minor: this seems like a good chunk of code to break into its own component! just trying to keep react components tighter in general.

Reply:
create new new component for Data product ID and PR update